src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -220,10 +248,11 @@ public static class StandardLearnersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer.
+        /// Predict a target using a linear binary classification model trained with averaged perceptron trainer using advanced options.
+        /// For usage details, please see <see cref="AveragedPerceptron(BinaryClassificationCatalog.BinaryClassificationTrainers, string, string, string, IClassificationLoss, float, bool, float, int)"/>
         /// </summary>
         /// <param name="catalog">The binary classification catalog trainer object.</param>
-        /// <param name="options">Advanced arguments to the algorithm.</param>
+        /// <param name="options">Advanced trainer options.</param>
         public static AveragedPerceptronTrainer AveragedPerceptron(
```

Comment:
this one would love sample too!
Especially how to change loss function, and change calibrator.

Reply:
done

---
In reply to: [255262343](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255262343) [](ancestors = 255262343)